ngraph/test/frontend/frontend_manager.cpp

```diff
@@ -141,12 +141,16 @@ TEST(FrontEndManagerTest, testDefaultPlace)
     ASSERT_ANY_THROW(place->get_names());
     ASSERT_ANY_THROW(place->get_consuming_operations());
     ASSERT_ANY_THROW(place->get_consuming_operations(0));
+    ASSERT_ANY_THROW(place->get_consuming_operations("", 0));
```

Comment:
also add `get_consuming_operations("")` to achieve 100% code coverage :-)

Reply:
done in https://github.com/openvinotoolkit/openvino/pull/6637